mineral-ui/mineral-ui
# [Table] Improve sorting

### Description

- Prevent Tables with a controlled `sort` from sorting the data internally (see breaking change, below)
- Update examples & docs necessarily
- Fix a small visual issue with sortable header cells where the dotted border was displaying over the focus outline

### Motivation and context

Closes #752 

### Screenshots, videos, or demo, if appropriate


https://752-sortable-improvements--mineral-ui.netlify.com/components/table/sortable-paginated

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to Table > [Sortable — Controlled](https://752-sortable-improvements--mineral-ui.netlify.com/components/table/sortable-controlled) & [Sortable — Paginated](https://752-sortable-improvements--mineral-ui.netlify.com/components/table/sortable-paginated) examples
3. Confirm it works properly

### Types of changes

- Breaking change (fix or feature that would cause existing functionality to change)

Breaking: Tables with a controlled `sort` will now need to sort their data and pass that sorted data to the Table

### Checklist
